### ISSUE TITLE
No channels listed with initial v1.0.1 installation

### ISSUE THREAD

**Issue by mzsombor** [cralin/plugin.video.DigiOnline.ro#4]
Hi,

I was unable to start the addon with an initial v1.0.1 installation. The channel list was empty and debug showed the following:

```

20200226_104912 <module> DEBUG: [ Addon settings ] _config_DebugEnabled_ = true
20200226_104912 <module> DEBUG: [ Addon settings ] _config_ShowTitleInChannelList_ = true
20200226_104912 <module> DEBUG: [ Addon cookiefile ] addon_cookiesfile_name = /home/zsombor/.kodi/userdata/addon_data/plugin.video.DigiOnline.ro/cookies.txt
20200226_104912 <module> DEBUG: [ Addon cookiejar ] Loaded cookiejar from file: /home/zsombor/.kodi/userdata/addon_data/plugin.video.DigiOnline.ro/cookies.txt
20200226_104912 <module> DEBUG: Enter function
20200226_104912 router DEBUG: Enter function
20200226_104912 check_defaults_DigiOnline_account DEBUG: Enter function
20200226_104912 check_defaults_DigiOnline_account DEBUG: Exit function
20200226_104912 list_categories DEBUG: Enter function
20200226_104912 list_categories INFO: Read categories from cache
20200226_104912 list_categories DEBUG: Exit function
20200226_104912 router DEBUG: Exit function
20200226_104912 <module> DEBUG: Exit function
```

Out of curiosity I tried with v1.0.0 which was working ok. After removing v1.0.0 and changing it back to v1.0.1, v1.0.1 started to work as well. I suppose something weird is happening with the initial caching option or maybe it is my setup.

Feel free to close this issue, I just wanted to notify you in case anyone else will face this problem.

**Comment by cralin** [comment]
Hi,

Thank you for pointing this out.
Yes, it looks like there is a problem there especially when starting with
1.0.1 since the cached data is not available at that time and what is being
read is an empty list.

I have ongoing work to code this caching in a better way so these
situations would not occur anymore in the future.

**Comment by cralin** [comment]
Hi,

Could you please take the v1.0.2, uninstall the 1.0.1 with removing also
the addon settings and check on your side as well for the issue reported
here ?

Thank you.

**Comment by cralin** [comment]
Fixed in v1.0.2

**Comment by mzsombor** [comment]
Hi @cralin ,

I just did a fresh install on a rpi3 osmc instance and I can confirm it is working. Thanks again, I guess we will keep in touch :D.
